assemblers/org.osbuild.qemu

```diff
@@ -561,15 +567,24 @@ def install_grub2(image: str, pt: PartitionTable, options):
     grub_path = os.path.relpath("/boot/grub2", boot_part.mountpoint)
 
     # now created the core image
-    subprocess.run(["grub2-mkimage",
-                    "--verbose",
-                    "--directory", f"/usr/lib/grub/{platform}",
-                    "--prefix", f"(,{partid})/{grub_path}",
-                    "--format", platform,
-                    "--compression", "auto",
-                    "--output", core_path] +
-                   modules,
-                   check=True)
+    subprocess.run(
+        [
+            "grub2-mkimage",
+            "--verbose",
+            "--directory",
+            f"/usr/lib/grub/{platform}",
+            "--prefix",
+            f"(,{partid})/{grub_path}",
+            "--format",
+            platform,
+            "--compression",
+            "auto",
+            "--output",
+            core_path,
+        ]
+        + modules,
+        check=True,
+    )
```

Comment:
Some of the changes are for the better (some functions with 2-3 args on multiple lines getting joined in 1 line is ok, for example) but every list of arguments like this becomes almost unreadable.  If we could disable this enforcement of lists being one element per line, it would go a long way towards making it more palatable.